src/Frontend/Pages/Products/Products.jsx

```diff
@@ -16,6 +21,13 @@ function Products() {
   useEffect(() => setProducts(filteredProducts), [filteredProducts]);
   useEffect(() => setProducts(products), [products]);
 
+  const handleAddToCart = (product) => {
+    cart.dispatch({ type: ADD_TO_CART, data: { product } });
```

Comment:
Need to fix : If a product is already in the cart, clicking on add to cart keeps increasing the count on the cart badge in navbar. Then the cart also shows the same item 5 times.

Reply:
Missed the edge case thanks for noticing will fix this